lib/presto-client/index.js

```diff
@@ -2,8 +2,8 @@ const { URL } = require('url') ;
 
 var adapterFor = (function() {
     var adapters = {
-        'http:': require('http'),
-        'https:': require('https'),
+        'http:': require('follow-redirects/http'),
+        'https:': require('follow-redirects/https'),
```

Comment:
Doing it this way is recommended if want to give end-users the ability to fallback to native modules for webpack and friends. See https://www.npmjs.com/package/follow-redirects#browser-usage for more details.